elementor/elementor
# Fix: Star Rating Widget Display [ED-7362]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* It's a display issue fix of Star Rating Widget

## Description
An explanation of what is done in this PR

* It's a display issue fix of Star Rating Widget, When you input a rating value that has two decimal places it rounds it up, while if you use one decimal place it works as intended and doesn't round up. I noticed this when I did a customers website and did an average rating of 4.65 and it rounded to 5 in star value. you can know more about this issue from here: https://github.com/elementor/elementor/issues/18169

## Test instructions
This PR can be tested by following these steps:

* Add star rating widget to a page. Add a value like '4.65' to the rating field then click the 'Style' tab and you will see it round up.

## Quality assurance


Fixes #
